### PR TITLE
add media settings and components

### PR DESCRIPTION
**Pull request by DavidCramer**

### CODE REVIEW

File: php/class-connect.php [hunk 1]
```diff
@@ -730,11 +730,33 @@ public function settings() {
...
+						'text'      => __( 'Configure how your images are shown on your site. You can apply transformations to adjust the quality, format or visual appearance and define other settings such as responsive images.', 'cloudinary' ),
+						'url'       => 'https://cloudinary.com/documentation/image_transformations#quick_example',
+						'link_text' => __( 'See Examples', 'cloudinary' ),
+					),
+					array(
+						'type'      => 'info_box',
+						'icon'      => $this->plugin->dir_url . 'css/video.svg',
+						'title'     => __( 'Video Settings', 'cloudinary' ),
+						'text'      => __( 'Configure how your videos are shown on your site. You can apply transformations to adjust the quality, format or visual appearance and define other settings such as whether to use the Cloudinary video player.', 'cloudinary' ),
+						'url'       => 'https://cloudinary.com/documentation/image_transformations#quick_example',
```
Comment (pereirinha) [inline php/class-connect.php:750]: @DavidCramer 

Let's update the link here to the right location.

File: php/class-connect.php [hunk 2]
```diff
@@ -730,11 +730,33 @@ public function settings() {
...
+						'link_text' => __( 'See Examples', 'cloudinary' ),
+					),
+					array(
+						'type'      => 'info_box',
+						'icon'      => $this->plugin->dir_url . 'css/transformation.svg',
+						'title'     => __( 'Learn More', 'cloudinary' ),
+						'text'      => __( 'You can upload and manage your images in Cloudinary directly from your WordPress interface. The plugin also supports automated (single-click) migration of all images from your existing posts to Cloudinary. Once your WordPress images are stored in Cloudinary, you can take advantage of Cloudinary\'s transformation, optimization, and responsive image features as well as fast CDN delivery.', 'cloudinary' ),
+						'url'       => 'https://cloudinary.com/documentation/image_transformations#quick_example',
```
Comment (pereirinha) [inline php/class-connect.php:758]: @DavidCramer 

Same here.

File: php/class-sync.php [hunk 3]
```diff
@@ -849,11 +849,33 @@ public function settings() {
...
 					'type'    => 'radio',
 					'title'   => __( 'Sync Method', 'cloudinary' ),
-					'slug'    => 'sync_method',
-					'default' => 'manual',
+					'slug'    => 'auto_sync',
+					'default' => 'off',
 					'options' => array(
-						'auto'   => __( 'Auto Sync', 'cloudinary' ),
-						'manual' => __( 'Manual Sync', 'cloudinary' ),
+						'on'   => __( 'Auto Sync', 'cloudinary' ),
```
Comment (pereirinha) [inline php/class-sync.php:855]: Minor alignment issue :)

**Review by pereirinha — COMMENTED** [review]
@DavidCramer 

I'm finding only minor things to be addressed. I also PM'd you on backchannel for other items that are more visual.

Thank you for the great work.

**Comment by pereirinha** [comment]
@DavidCramer 

When possible, can you please resolve the conflicts, and merge this as is?

**Comment by DavidCramer** [comment]
@pereirinha - will do. It looks like it's just the build, and those changes to the text component.

I'll resolve and merge asap.

**Pull request merged**